src/models/network.js

```diff
@@ -189,7 +189,7 @@ Network.prototype.createIrcFramework = function (client) {
 
 	// Request only new messages from ZNC if we have sqlite logging enabled
 	// See http://wiki.znc.in/Playback
-	if (client.config.log && client.messageStorage.find((s) => s.canProvideMessages())) {
+	if (client.messageProvider) {
```

Comment:
Revert this so we maintain support for multiple message storage plugins

Reply:
Refer to https://github.com/thelounge/thelounge/pull/4197#discussion_r612052240